printer_mods/Baltojikale/LoudOwl aka Stabby/README.md

```diff
@@ -0,0 +1,26 @@
+# LoudOwl aka Stabby
+
+<img src="Visual/LoudOwl_2.jpg" width="400px" />
+<img src="Visual/LoudOWL.jpg" width="400px" />
+
+This is alternative part cooling fan solution which uses dual 5015 fan setup for Afterburner Toolhead. Replaces original parts that houses 4020 fan or any other similar type of mod. This mod is more of situational add on for PLA or similar type filaments which might require a lot of cooling (or printing at super high speeds). It is not constant solution as it increases gantry weight.
+
+## Hardware
+* 2x 5015 24v fans
+* 1x M3x20 SCHS (DIN912)
+
+## Printing
+
+Front piece and Logo piece requires supports!!!!!!!! Used recommended settings for Voron parts: 4 perimeters, 5 tops/bottoms, 40% infill. Though it is possible to use less plastic to reduce weight of plastic parts: 3 perimeters, 3 tops bottoms, 25% infill.
```

Comment:
We cannot merge STLs that require supports.  Please redesign the parts so that no support is required.  If using supports is unavoidable, they must be built-in to the STL with clear instructions in the README to remove the built-in supports after printing.

Reply:
Back  piece dont have those overhangs anymore as geometry was solved by introducing chamfers where necessary. Front piece and logo piece now have alternative versions with in build supports. They were tested by printing and successfully removing them